draft/draft-ietf-ippm-twamp-yang.xml

```diff
@@ -486,8 +486,8 @@ o-------------------o                       o-------------------o
         the operator of this Control-Client, including the authentication and
         encryption Modes. Specifically, mode-preference-chain lists the mode
         and its corresponding priority, expressed as a 16-bit unsigned
-        integer, where zero is the highest priority and subsequent values are
-        monotonically increasing.</t>
+        integer, where zero is the highest priority and subsequent values
+        integers increase by one.</t>
```

Comment:
I don't get "...and subsequent values integers increase by one.", isn't "where zero is the highest priority." clear enough?